src/components/manage/Preferences/PersonalInformation.jsx

```diff
@@ -99,9 +99,22 @@ class PersonalInformation extends Component {
    * @returns {string} Markup for the component.
    */
   render() {
+    if (this.props.error && this.props.error.status === 401) {
+      return <Unauthorized />;
+    }
+
+    const title = this.props.match.params.username
+      ? `Personal Information for ${
```

Comment:
This should use `<FormattedMessage ..>` with the relevant parameters, to be able to translate it right?